MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1275,12 +1283,34 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
         panTech.add(choiceTechLevel, gridBagConstraints);
 
+        variableTechLevelBox.setText(resourceMap.getString("variableTechLevelBox.text")); // NOI18N
+        variableTechLevelBox.setToolTipText(resourceMap.getString("variableTechLevelBox.toolTipText")); // NOI18N
+        variableTechLevelBox.setName("variableTechLevelBox"); // NOI18N
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 7;
+        gridBagConstraints.gridwidth = 2;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.HORIZONTAL;
+        gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+        panTech.add(variableTechLevelBox, gridBagConstraints);
+
+        factionIntroDateBox.setText(resourceMap.getString("factionIntroDateBox.text")); // NOI18N
+        factionIntroDateBox.setToolTipText(resourceMap.getString("factionIntroDateBox.toolTipText")); // NOI18N
+        factionIntroDateBox.setName("factionIntroDateBox"); // NOI18N
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 8;
+        gridBagConstraints.gridwidth = 2;
```

Comment:
The previous control was at gridy 7 with a width of 2 and this is on gridy 8 also with a width of 2.  Shouldn't this one be on gridy 9 to avoid overlap?

Reply:
width affects horizontal spacing. gridy is vertical positioning.

Reply:
Gah, you're right.